docs/guide/counter/plugins/counter/counter.go

```diff
@@ -1,98 +1,197 @@
 package counter
 
 import (
-	"fmt"
+	rawerr "errors"
```

Comment:
can we try to have the `errors` package or `github.com/pkg/errors` package have the original `errors` reference namespace and change anything else to a custom name? can just be confusing. This is more of a consistency thing - does this make sense? Also, don't know if this applies here but sometimes I use `fmt.Errorf` to avoid importing `errors` hehe